addons/l10n_es_edi_sii/data/template/account.tax-es_common.csv

```diff
@@ -1,7 +1,7 @@
 "id","name","description","invoice_label","l10n_es_type","l10n_es_bien_inversion","l10n_es_exempt_reason","tax_scope"
 "account_tax_template_s_iva21b","","","","sujeto","","","consu"
 "account_tax_template_s_iva21s","","","","sujeto","","","service"
-"account_tax_template_s_iva21isp","","","","sujeto_isp","","",""
+"account_tax_template_s_iva21isp","","","","sujeto","","",""

```

Comment:
This doesn't work.
the name is required on `account.tax`.

Reply:
@KangOl It is set in l10n_es (The name was done double to give a bit of context about the tax, but the best idea might be to actually remove the column here)
![Screenshot from 2023-05-15 22-18-28](https://github.com/odoo/odoo/assets/3832699/f78069ab-cb7a-46ab-ab8e-f9054788bdde)

Reply:
No, it's not. The xmlid point to the current module. And even if it wasn't, it would reset the name to an empty string.

Reply:
@KangOl It works a bit different with the template logic added in saas-16.2

Reply:
It doesn't change the fact that this module is not installable.

Reply:
Here: https://upgradeci.odoo.com/upgradeci/run/71626